lib/rucio/web/rest/flaskapi/v1/auth.py

```diff
@@ -1444,7 +1444,7 @@ def get(self) -> 'ResponseReturnValue':
         headers.set('Pragma', 'no-cache')
 
         if not EXTRA_MODULES['onelogin']:
-            return "SAML not configured on the server side.", 400, headers
+            return generate_http_error_flask(status_code=400, exc="ConfigurationError", exc_msg="SAML not configured on the server side.", headers=headers)
```

Comment:
```suggestion
            return generate_http_error_flask(status_code=400, exc=ConfigurationError.__name__, exc_msg="SAML not configured on the server side.", headers=headers)
```

for consistency with most other `generate_http_error_flask` references, where we pass the exc name via `__name__`